pytket/phir/phirgen_parallel.py

```diff
@@ -245,6 +245,10 @@ def format_and_add_primary_commands(
                     pc = shard.primary_command
                     fmt_measure["args"].append(arg_to_bit(pc.args[0]))
                     fmt_measure["returns"].append(arg_to_bit(pc.bits[0]))
+                # sort to ensure deterministic order of qbuits/cbits in output
+                # lowest -> highest
+                fmt_measure["args"].sort()
+                fmt_measure["returns"].sort()
```

Comment:
I am confident that it will. The args/bits are always in this format ['q' or 'c', number] and it sorts by the number. As long as that format doesn't change it should work

Reply:
Thanks, removing this commit from this PR just to keep things separated.